apps/storybook/README.md

```diff
@@ -0,0 +1,5 @@
+## NameKit React storybook documentation
+
+### Styling with TailwindCSS
+
+In `package.json`, a command called `tailwind` is defined: this command creates an `output.css` file which basically contains Tailwind CSS specs that are used whenever a Tailwind class is referenced! This `output.css` file is imported in `/.storybook/preview.js` and, this way, makes Tailwind specs available throughout the Storybook context. This command is also referenced in the `build` command since it is essential that our Tailwind specs are up to date to the Tailwind version we have imported in the app.
```

Comment:
@lightwalker-eth @notrab since I have removed Tailwind from this codebase, as per an alignment with @notrab, this README will be changed to not include the above info!